benchmarks/invert.bench.ts

```diff
@@ -0,0 +1,18 @@
+import { bench, describe } from 'vitest';
+import { invert as invertByLodash } from 'lodash';
+import { invert as invertByToolkit } from 'es-toolkit';
+
+const object: { [key: string]: string } = {};
+for (let i = 0; i < 10000; i++) {
+  object[`key${i}`] = `value${i}`;
+}
+
+describe('invert function benchmark', () => {
+  bench('Lodash invert', () => {
+    invertByLodash(object);
+  });
+
+  bench('Custom invert', () => {
```

Comment:
**custom** is good, but what do you think about doing it with `es-toolkit/invert` ?

Reply:
@hautest 
Thank you for the good comments, I've made the corrections more clearly!

Reply:
@hautest 
Thank you for the good comments, I've made the corrections more clearly!